generators/client/templates/react/package.json.ejs

```diff
@@ -26,138 +26,138 @@
+    "autoprefixer": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "browser-sync": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "browser-sync-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "cache-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (protractorTests) { _%>
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     <%_ } _%>
-    "copy-webpack-plugin": "6.0.3",
-    "core-js": "3.6.5",
-    "cross-env": "7.0.2",
-    "css-loader": "3.5.3",
-    "eslint": "7.13.0",
-    "eslint-config-prettier": "6.15.0",
-    "eslint-plugin-react": "7.21.5",
-    "eslint-webpack-plugin": "2.3.0",
-    "file-loader": "6.0.0",
-    "fork-ts-checker-webpack-plugin": "4.1.3",
-    "friendly-errors-webpack-plugin": "1.7.0",
+    "copy-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "core-js": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "cross-env": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "css-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "eslint": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "eslint-config-prettier": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "eslint-plugin-react": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "eslint-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "file-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "fork-ts-checker-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "friendly-errors-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (!skipJhipsterDependencies) { _%>
     "generator-jhipster": "<%= packagejs.version %>",
         <%_ otherModules.forEach(module => { _%>
     "<%= module.name %>": "<%= module.version %>",
         <%_ });
     } _%>
-    "html-webpack-plugin": "4.3.0",
-    "identity-obj-proxy": "3.0.0",
-    "jest": "26.4.2",
-    "jest-junit": "11.1.0",
-    "jest-sonar-reporter": "2.0.0",
-    "json-loader": "0.5.7",
+    "html-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "identity-obj-proxy": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "jest": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "jest-junit": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "jest-sonar-reporter": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "json-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (!skipCommitHook) { _%>
     "husky": "VERSION_MANAGED_BY_CLIENT_COMMON",
     "lint-staged": "VERSION_MANAGED_BY_CLIENT_COMMON",
     <%_ } _%>
     <%_ if (enableTranslation) { _%>
-    "merge-jsons-webpack-plugin": "1.0.21",
+    "merge-jsons-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ } _%>
-    "mini-css-extract-plugin": "0.9.0",
+    "mini-css-extract-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
     <%_ if (protractorTests) { _%>
     "mocha": "7.1.2",
     <%_ } _%>
-    "optimize-css-assets-webpack-plugin": "5.0.3",
-    "postcss-loader": "3.0.0",
+    "optimize-css-assets-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_REACT",
+    "postcss-loader": "VERSION_MANAGED_BY_CLIENT_REACT",
     "prettier": "<%= PRETTIER_VERSION %>",
```

Comment:
Add prettier and prettier Java to dependabot

Reply:
ok